cobbler.spec

```diff
@@ -206,14 +206,14 @@ Requires:       xorriso
 Requires:       %{py3_module_cheetah}
 Requires:       %{py3_module_dns}
 Requires:       python%{python3_pkgversion}-future
-Requires:       python%{python3_pkgversion}-ldap3
 Requires:       %{apache_mod_wsgi}
 Requires:       python%{python3_pkgversion}-netaddr
 Requires:       %{py3_module_pyyaml}
 Requires:       python%{python3_pkgversion}-requests
 Requires:       python%{python3_pkgversion}-simplejson
 Requires:       python%{python3_pkgversion}-tornado
 Requires:       python%{python3_pkgversion}-distro
+Recommends:     python%{python3_pkgversion}-ldap3
```

Comment:
cobbler will crash without this module installed.

Reply:
Also, you cannot use `Recommends` on EL7.

Reply:
Indeed, undo this change.

Reply:
Not sure about this. We had it running on a Jeos installation without this pkg. Where would it crash? I can seen an ldap authentication moduel. But if you are not using that, you should be fine. But should of course be gated for EL7.

Reply:
Hmm, I suppose as long as we're not using any setuptools generated entry point files, it wouldn't be noticed as much. But any time code that would use it would cause it to crash, or if the import was hit for any reason. In general, if it's in `install_requires` it should be installed.

Reply:
@Conan-Kudo You`re right. It makes sense to leave it as "required" since it is listed in "install_requires" in the setup.py.